Makefile

```diff
@@ -261,4 +255,16 @@ ci-verify:
 
 .NOTPARALLEL: ember-dist ember-dist-dev
 
--include packagespec.mk
+.PHONY: build
```

Comment:
Maybe add a comment noting that this target is intended specifically for release builds?